pandas-stubs/io/xml.pyi

```diff
@@ -15,6 +15,7 @@ from pandas._typing import (
 
 def read_xml(
     path_or_buffer: FilePath | ReadBuffer[bytes] | ReadBuffer[str],
+    *,
```

Comment:
Not clear why you added this??

Reply:
We deprecated passing positional arguments to read_xml in 1.4

Deprecated passing arguments as positional for [read_xml()](https://pandas.pydata.org/pandas-docs/version/1.5.0rc0/reference/api/pandas.read_xml.html#pandas.read_xml) other than path_or_buffer ([GH45133](https://github.com/pandas-dev/pandas/issues/45133))

Reply:
OK, the docs don't say anything about that, so maybe the docs should be updated as well.

Reply:
I think we are not mentioning positional deprecations in the docs